src/components/SprCard.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+    Box,
+    Flex,
+    Stat,
+    StatArrow,
+    StatHelpText,
+    StatLabel,
+    Tooltip,
+} from '@chakra-ui/react';
+import { Player } from '../types/domain/Player';
+import { SprTag } from './SprTag';
+
+export const SprCard = ({
+    player,
+    sprTrend,
+}: {
+    player: Player;
+    sprTrend: number;
+}) => {
+    return (
+        <Flex direction='column' align='center'>
+            <Box>
+                <Stat>
+                    <Flex direction='column' align='center'>
+                        <SprTag props={{ size: 'xl' }} player={player} />
+                        <StatLabel fontSize='20'>
+                            Season Power Ranking
```

Comment:
Should we make this just say "SPR" instead of "Season Power Ranking" since the hover text explains what it means anyway?

Reply:
Honestly I just like the whole text "Season Power Ranking" more simply because I think it looks better
But I can go either way on it

![image](https://user-images.githubusercontent.com/34924609/197253054-9419b4b4-a3ae-4836-aa72-7ccb5d175fc0.png)

![image](https://user-images.githubusercontent.com/34924609/197253135-8b06da99-fbe4-4803-a1c8-329cc56aa78c.png)

Whatcha think @davidpchi ?